mslib/msui/mscolab.py

```diff
@@ -667,14 +670,29 @@ def fetch_gravatar(self, refresh=False):
                 return
 
         if refresh and not email_in_config:
-            show_popup(
-                self.prof_diag,
-                "Information",
+            update_messagebox = QtWidgets.QMessageBox.question(
+                None, "Information",
                 "Please add your email to the gravatar_ids section in your "
                 "msui_settings.json to automatically fetch your gravatar",
-                icon=1, )
-
-        self.set_gravatar(gravatar_img_path)
+                QtWidgets.QMessageBox.Open | QtWidgets.QMessageBox.Ok)
+            if update_messagebox == QtWidgets.QMessageBox.Open:
+                config_location = constants.MSUI_CONFIG_PATH
+                path = os.path.join(config_location, "msui_settings.json")
```

Comment:
when the "msui_settings.json" please use

from mslib.msui.constants import MSUI_SETTINGS